tests/utils/k8s.go

```diff
@@ -161,6 +162,59 @@ func waitForPodStatusPhase(clientSet *kubernetes.Clientset, podName string, name
 	})
 }
 
+func EnvVarInPod(envVar v1.EnvVar, pod v1.Pod) bool {
```

Comment:
The purpose of the function is not clear from its name. Can we rename it to something like `IsEnvVarPresentInPod`?